spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -221,6 +221,8 @@ class CodeEditor(TextEditBaseWidget):
 
     sig_breakpoints_changed = Signal()
     sig_debug_stop = Signal(int)
+    sig_stop_debugging = Signal()
+    sig_start_debugging = Signal()

```

Comment:
Let's change this to `sig_debug_start = Signal()` for consistency with `sig_debug_stop`.